metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -237,6 +246,16 @@ func CreateStackdriverSink(uri *url.URL) (core.DataSink, error) {
 		cluster_name = opts["cluster_name"][0]
 	}
 
+	minInterval := time.Nanosecond
```

Comment:
This is an explicit default in go, e.g. linter will ask to change `var x int = 0` to `var x int`. I don't like 1 nanosecond as an initial value, but leave it up to you